content/blog/what’s-new-with-chapel-nine-questions-for-the-development-team.md

```diff
@@ -0,0 +1,17 @@
+---
+title: What’s New with Chapel? Nine Questions for the Development Team
+date: 2024-09-04T16:54:10.515Z
+featuredBlog: true
+externalLink: https://www.hpcwire.com/2024/09/04/whats-new-with-chapel-nine-questions-for-the-development-team/
+author: HPE DEV
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - Chapel
```

Comment:
You can remove "Chapel" (with capital C) as tag. Just keep the tag: "chapel" with (lowercase "c").

Reply:
Done—thanks for letting me know that!  I'm not sure how I got into this pattern, but know I've been doing it since virtually the outset and feeling weird about it…